phys/module_physics_init.F

```diff
@@ -3610,6 +3613,24 @@ SUBROUTINE bl_init(STEPBL,BLDT,DT,RUBLTEN,RVBLTEN,RTHBLTEN,  &
                 CALL wrf_error_fatal ( 'arguments not present for calling TEMF scheme' )
          ENDIF
 
+      CASE (EEPSSCHEME)
+           IF(isfc .ne. 1 .and. isfc .ne. 2 .and. isfc .ne. 5 ) CALL wrf_error_fatal &
```

Comment:
@ChunxiZhang-NOAA The old MM5 surface layer scheme should also work. That option is 91 now.

Reply:
Agreed.

Reply:
@ChunxiZhang-NOAA Can you make the change then?

Reply:
@ChunxiZhang-NOAA @weiwangncar 
The IF test needs to include isfc==91.